documentation/assemblies/operators/assembly-using-the-topic-operator.adoc

```diff
@@ -24,8 +24,6 @@ include::../../modules/operators/proc-converting-managed-topics.adoc[leveloffset
 include::../../modules/operators/proc-converting-non-managed-topics.adoc[leveloffset=+1]
 //deleting managed topics
 include::../../modules/operators/con-deleting-managed-topics.adoc[leveloffset=+1]
-//changing the topic operator mode
-include::../../modules/operators/proc-changing-topic-operator-mode.adoc[leveloffset=+1]
```

Comment:
we don't want to show this even for upgrades and downgrades?

Reply:
Not sure about that. Maybe only from BTO to UTO in the context of upgrades?

@ppatierno @scholzj wdyt?

Reply:
Not sure about the relevance -> there is no choice anymore as we have only UTO in 0.41.